SYCL/ESIMD/dpas/dpas_int.cpp

```diff
@@ -0,0 +1,58 @@
+//==---------------- dpas_int.cpp  - DPC++ ESIMD on-device test ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies DPAS support for 2,4,8-bit integers.
+
+#include "dpas_common.hpp"
```

Comment:
Nit: just curious - why not simply template by element types in a single test source instead of including common header?

Reply:
just wanted to have different tests for different groups of types: int separate, fp16 - separate, etc.
Otherwise, it would be too big test running too long. 
dpas_int.cpp already has too many test cases in it. And it needs some more extension to test the cases when precision is not specified.